packages/callbacks/tizen/inc/macros.h

```diff
@@ -6,379 +6,430 @@
-      cv.notify_one();                                                      \
-    };                                                                      \
-                                                                            \
-    CallbackWrapper *wrapper_ptr = new CallbackWrapper(wrapper);            \
-    LDEBUG("calling RequestCallbackCall()");                                \
-    RequestCallbackCall(wrapper_ptr);                                       \
-                                                                            \
-    LDEBUG("waiting for notification");                                     \
-    while (!dartCallbackFinished) {                                         \
-      cv.wait(lock);                                                        \
-    }                                                                       \
-    LDEBUG("exit, dartCallbackRetVal: %d", (int)dartCallbackRetVal);        \
-    return dartCallbackRetVal;                                              \
-  }                                                                         \
-                                                                            \
-  CB_RETURN platform_blocking_##CB_NAME##_0(CB_PARAMS) {                    \
-    return platform_blocking_##CB_NAME(EXPAND_LIST(CB_PARAMS_NAMES));       \
-  }                                                                         \
-  CB_RETURN platform_blocking_##CB_NAME##_1(CB_PARAMS) {                    \
-    return platform_blocking_##CB_NAME(EXPAND_LIST(CB_PARAMS_NAMES));       \
-  }                                                                         \
-  CB_RETURN platform_blocking_##CB_NAME##_2(CB_PARAMS) {                    \
-    return platform_blocking_##CB_NAME(EXPAND_LIST(CB_PARAMS_NAMES));       \
-  }                                                                         \
-  CB_RETURN platform_blocking_##CB_NAME##_3(CB_PARAMS) {                    \
-    return platform_blocking_##CB_NAME(EXPAND_LIST(CB_PARAMS_NAMES));       \
-  }                                                                         \
-  CB_RETURN platform_blocking_##CB_NAME##_4(CB_PARAMS) {                    \
-    return platform_blocking_##CB_NAME(EXPAND_LIST(CB_PARAMS_NAMES));       \
+#define PROXY_GROUP_RETURN(CB_NAME, CB_RETURN, CB_PARAMS...)                 \
+  CB_RETURN platform_blocking_##CB_NAME(CB_PARAMS) {                         \
+    if (INVALID_USER_DATA(user_data)) {                                      \
+      LOG_ERROR("Received bad user_data in %s: %p - ignoring", #CB_NAME,     \
+                user_data);                                                  \
+      return CB_RETURN();                                                    \
+    }                                                                        \
+    uint32_t _callback_id = GET_CALLBACK_ID_FROM_USER_DATA(user_data);       \
```

Comment:
Underscores (`_` and `__`) are not expected before variable names.

https://google.github.io/styleguide/cppguide.html#Naming

Reply:
Hmm, if that's really the case, prefixing with underscores should be fine.